### ISSUE TITLE
aws_cloudformation_stack_set has no field to set max_concurrent operations

### ISSUE THREAD

**Issue by nandac** [hashicorp/terraform-provider-aws#9174]
Hi,

I have been experimenting with the aws_cloudformation_stack_set resource but find that there is no way to set the number of concurrent executions allowed. As a result I ran into an issue when creating stack set instances as terraform by default tries to create 10 resources in parallel.

Because the max concurrent executions number is set to 1 on the stack set I get:
```
Error: error creating CloudFormation Stack Set Instance: OperationInProgressException: Another Operation on StackSet arn:aws:cloudformation:<region>:<account>:stackset/<stack-name> is in progress
        status code: 409, request id: 2e0bf029-9923-11e9-9d56-6dd65b781a5f
```

I can get around the issue by running apply like this:
```
terraform apply -parallelism=1
```

I can set the max concurrent executions for the stack set in CloudFormation or with an API call so I believe the aws provider should have this feature as well.

Can you suggest a workaround?

Many thanks,
Nanda

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add a field to the aws_cloudformation_stack_set resource to set the maximum number of concurrent executions.

### New or Affected Resource(s)

* aws_cloudformation_stack_set

### References
Have not found any related issues.

**Comment by nijave** [comment]
It seems like a sensible solution to this would be for Terraform to treat 409 responses as "throttling" and retry later

**Comment by nijave** [comment]
https://github.com/terraform-providers/terraform-provider-aws/pull/10969

**Comment by breathingdust** [comment]
Looks like this was resolved in #10969
